cpuid_arm64.c

```diff
@@ -41,6 +41,8 @@
 #define CPU_THUNDERX2T99  8
 //Hisilicon
 #define CPU_TSV110        9
+// Ampere
+#define CPU_EMAG8180	 10
```

Comment:
Thanks @martin-frbg. Do we want to use the specific name or the microarchitecture name `Skylark`?

Reply:
Well, Skylark certainly sounds more poetic than EMAG8180, but I guess specific name would be better if there could be any related chips with different cache sizes ?